task/trigger-jenkins-job/0.1/tests/pre-apply-task-hook.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# This will create a Jenkins deployment, a jenkins service, grab the password,
+# gets a 'crumb' and generate a secret out of it for our task to run against to.
+
+cat <<EOF | kubectl apply -f- -n "${tns}"
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jenkins
+spec:
+  selector:
+    matchLabels:
+      run: jenkins
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        run: jenkins
+    spec:
+      containers:
+      - name: jenkins
+        image: jenkins/jenkins:lts
+        ports:
+        - containerPort: 8080
```

Comment:
CI is failing on OpenShift cluster. Probable fix would be adding a `volumeMount` in `containerSpec`
```suggestion
      containers:
      - name: jenkins
        image: jenkins/jenkins:lts
        ports:
        - containerPort: 8080
        volumeMounts:
          - name: jenkins-home
            mountPath: /var/jenkins_home
      volumes:
        - name: jenkins-home
          emptyDir: {}
```

Reply:
Nicely found! thanks!